packages/thirdweb/src/wallets/wallet-connect/index.ts

```diff
@@ -499,6 +509,13 @@ function getChainsToRequest(options: {
       ? (optionalChainIds as ArrayOneOrMore<number>)
       : [1];
 
+  if (!options.chain && optionalChains.length === 0) {
+    rpcMap[1] = getRpcUrlForChain({
```

Comment:
should use getCachedChain here i think otherwise it wont respect the users RPC override

Reply:
done

Reply:
done